docs/tutorial.rst

```diff
@@ -121,22 +125,26 @@ There is a preferred way to resolve this. If any kind of fitting object has been
 containing an instance of ``Model``. This can again be called::
 
     model = {y_1: a * x**2, y_2: 2 * x * b}
-    fit = Fit(model, x=xdata)
+    fit = Fit(model, x=xdata, y_1=y_data1, y_2=y_data2)
     fit_result = fit.execute()
 
     y_1, y_2 = fit.model(x=xdata, **fit_result.params)
 
-This returns a tuple with the components evaluated so through the magic of tuple unpacking``y_1`` and ``y_2`` contain the
-evaluated fit. Nice!
+This returns a named tuple with the components evaluated so through the magic of tuple unpackingi ``y_1`` and ``y_2`` contain the
+evaluated fit. Nice! It is usually beter to do the unpacking explicitely though.
```

Comment:
I agree with myself (A).
Explicit is better than implicit. And I think it would be good to show both methods. I changed the wording to be less strong.